docker-compose.yml

```diff
@@ -10,14 +10,11 @@ services:
 
   ckan:
     container_name: ckan
-    image: adr.ckan:${IMAGE_TAG}
+    image: fjelltopp/ckan:development_base
```

Comment:
In this new approach how can we keep the option to quickly switch between different feature branches (braches which includes changes to the Pipfiles). Is it possible to configure which venv to use via env var?
Also the README section `## Docker images for feature branches` should be updated with new approach

Reply:
Single base image without the CKAN codebase, switching between feature branches is going to use same venv, `pipenv --sync` should handle that quickly enough